docs/concepts/redirects.mdx

```diff
@@ -40,105 +38,125 @@ will fail, however, for https://anotherapp.com will succeed on any path.
 Ory Cloud has a total of six flows, **Login**, **Registration**,
 **Verification**, **Recovery**, **Settings** and **Logout** which can be
 configured to redirect back to any URL. A common use case would be to redirect
-the user to your application home screen after a logout or to a specific url on
-a sub-domain after a settings password update. On project creation the redirect
-flows are set to the default Ory Cloud Managed UI. For more information on the
-Ory Managed UI, please see the [Managed User Interface](./managed-ui.mdx)
-documentation.
+the user to your application home screen after a logout or to a specific URL on
+a sub-domain after a settings password update.
 
-### Login, Registration and Settings
+On project creation the redirect flows are set to the default
+[Managed User Interface](./managed-ui.mdx).
+
+### Login, Registration, and Settings
 
 The Login and Registration flows have three fields (all optional):
 
 1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-OIDC Redirect URL` (**not supported**)
+2. `Post-Password Redirect URL`
+3. `Post-OIDC Redirect URL`
 
-And Settings has three fields (all optional):
+The Settings flow has three fields (all optional):
 
 1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-Profile Redirect URL`
-
-:::info
-
-Setting `Post-OIDC` redirects for Login and Registration will be supported in a
-future release. For more information see the
-[Ory Cloud Roadmap](../cloud/cloud-roadmap.mdx).
-
-:::
-
-When setting the `Default Redirect URL` as an FQDN (fully qualified domain name)
-such as `https://domain.example/login` it will overwrite the
-`Global Redirect URL`. As a path it will replace the `Global Redirect URL` path,
-for example a path such as `/login` will alter
-`https://domain.example/global/path` to `https://domain.example/login`.
-
-On the sub-conditions such as `Post-Password` or `Post-OIDC` an FQDN or a path
-can be set. In the case of an FQDN it will replace the closest relative starting
+2. `Post-Password Redirect URL`
+3. `Post-Profile Redirect URL`
+
+To overwrite the `Global Redirect URL` set the `Default Redirect URL` to a fully
+qualified domain name
+([FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) - for example
+`https://domain.example/login`).  
+To replace the `Global Redirect URL` path set the `Default Redirect URL` to a
+path. For example when setting it to `/login`,
+`https://domain.example/global/path` becomes `https://domain.example/login`.
```

Comment:
I don't think this works any more, can you confirm that this is true? I think you now just need to always specify the FQDN if it's not your CNAME or hosted URL or Ory Proxy base URL